src/renderer/families/cosmos/DelegationFlowModal/components/ValidatorRow.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+import { getDefaultExplorerView, getAddressExplorer } from "@ledgerhq/live-common/lib/explorers";
+import { LEDGER_VALIDATOR_ADDRESS } from "@ledgerhq/live-common/lib/families/cosmos/utils";
+import type { CosmosValidatorItem } from "@ledgerhq/live-common/lib/families/cosmos/types";
+import type { CryptoCurrency, Unit } from "@ledgerhq/live-common/lib/types";
+
+import LedgerLiveLogo from "~/renderer/components/LedgerLiveLogo";
+import Logo from "~/renderer/icons/Logo";
+
+import { BigNumber } from "bignumber.js";
+import React, { useCallback } from "react";
+import { Trans } from "react-i18next";
+import styled from "styled-components";
+import { urls } from "~/config/urls";
+import Box from "~/renderer/components/Box";
+import type { ValidatorRowProps } from "~/renderer/components/Delegation/ValidatorRow";
+import ValidatorRow, { IconContainer } from "~/renderer/components/Delegation/ValidatorRow";
+import FirstLetterIcon from "~/renderer/components/FirstLetterIcon";
+import Text from "~/renderer/components/Text";
+import Check from "~/renderer/icons/Check";
+import { openURL } from "~/renderer/linking";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+type Props = {
+  currency: CryptoCurrency,
+  validator: CosmosValidatorItem,
+  active?: boolean,
+  onClick?: (v: CosmosValidatorItem) => void,
+  unit: Unit,
+};
+
+function CosmosValidatorRow({ validator, active, onClick, unit, currency }: Props) {
+  const explorerView = getDefaultExplorerView(currency);
+  const onExternalLink = useCallback(
+    (address: string) => {
+      if (validator.validatorAddress === LEDGER_VALIDATOR_ADDRESS) {
+        openURL(urls.ledgerValidator);
+      } else {
+        const srURL = explorerView && getAddressExplorer(explorerView, address);
```

Comment:
What does srURL mean? ^^

Reply:
No idea, it's something that was used on the code I just copy it

Reply:
:man_shrugging: